test/app-tap/datetime.test.lua

```diff
@@ -16,15 +16,30 @@ local MAX_DATE_YEAR = 5879611
 local MAX_DATE_MONTH = 7
 local MAX_DATE_DAY = 11
 
-local incompat_types = 'incompatible types for comparison'
+local SECS_PER_DAY      = 86400
+local AVERAGE_DAYS_YEAR = 365.25
+local AVERAGE_WEEK_YEAR = (AVERAGE_DAYS_YEAR / 7)
+local MAX_YEAR_RANGE = (MAX_DATE_YEAR - MIN_DATE_YEAR)
+local MAX_MONTH_RANGE = (MAX_YEAR_RANGE * 12)
+local MAX_WEEK_RANGE = (MAX_YEAR_RANGE * AVERAGE_WEEK_YEAR)
+local MAX_DAY_RANGE = (MAX_YEAR_RANGE * AVERAGE_DAYS_YEAR)
+local MAX_HOUR_RANGE = (MAX_DAY_RANGE * 24)
+local MAX_MIN_RANGE = (MAX_HOUR_RANGE * 60)
+local MAX_SEC_RANGE = (MAX_DAY_RANGE * SECS_PER_DAY)
```

Comment:
You don't need `()` around right side of assignment operator. Here and in all other places.

Reply:
Removed:
```diff
--- a/src/lua/datetime.lua
+++ b/src/lua/datetime.lua
@@ -103,8 +103,8 @@ local MAX_DATE_DAY = 11
 -- 365 1/4 days = 365.25 days. This gives an error of
 -- about 1 day in 128 years.
 local AVERAGE_DAYS_YEAR = 365.25
-local AVERAGE_DAYS_MONTH = (AVERAGE_DAYS_YEAR / 12)
-local AVERAGE_WEEK_YEAR = (AVERAGE_DAYS_YEAR / 7)
+local AVERAGE_DAYS_MONTH = AVERAGE_DAYS_YEAR / 12
+local AVERAGE_WEEK_YEAR = AVERAGE_DAYS_YEAR / 7
 local INT_MAX = 2147483647
 local INT_MIN = -2147483648
 -- -5879610-06-22
@@ -113,13 +113,13 @@ local MIN_DATE_TEXT = ('%d-%02d-%02d'):format(MIN_DATE_YEAR, MIN_DATE_MONTH,
 -- 5879611-07-11
 local MAX_DATE_TEXT = ('%d-%02d-%02d'):format(MAX_DATE_YEAR, MAX_DATE_MONTH,
                                               MAX_DATE_DAY)
-local MAX_YEAR_RANGE = (MAX_DATE_YEAR - MIN_DATE_YEAR)
-local MAX_MONTH_RANGE = (MAX_YEAR_RANGE * 12)
-local MAX_WEEK_RANGE = (MAX_YEAR_RANGE * AVERAGE_WEEK_YEAR)
-local MAX_DAY_RANGE = (MAX_YEAR_RANGE * AVERAGE_DAYS_YEAR)
-local MAX_HOUR_RANGE = (MAX_DAY_RANGE * 24)
-local MAX_MIN_RANGE = (MAX_HOUR_RANGE * 60)
-local MAX_SEC_RANGE = (MAX_DAY_RANGE * SECS_PER_DAY)
+local MAX_YEAR_RANGE = MAX_DATE_YEAR - MIN_DATE_YEAR
+local MAX_MONTH_RANGE = MAX_YEAR_RANGE * 12
+local MAX_WEEK_RANGE = MAX_YEAR_RANGE * AVERAGE_WEEK_YEAR
+local MAX_DAY_RANGE = MAX_YEAR_RANGE * AVERAGE_DAYS_YEAR
+local MAX_HOUR_RANGE = MAX_DAY_RANGE * 24
+local MAX_MIN_RANGE = MAX_HOUR_RANGE * 60
+local MAX_SEC_RANGE = MAX_DAY_RANGE * SECS_PER_DAY
 local DEF_DT_ADJUST = builtin.DT_LIMIT
 
 local date_tostr_stash =
@@ -1126,16 +1126,16 @@ local interval_index_fields = {
     msec = function(self) return math_floor(self.nsec / 1e6) end,
 
     week = function(self)
-        return (math_modf(total_secs(self) / (7 * SECS_PER_DAY)))
+        return math_modf(total_secs(self) / (7 * SECS_PER_DAY))
     end,
     day = function(self)
-        return (math_modf(total_secs(self) / SECS_PER_DAY))
+        return math_modf(total_secs(self) / SECS_PER_DAY)
     end,
     hour = function(self)
-        return (math_modf(total_secs(self) / (60 * 60)))
+        return math_modf(total_secs(self) / (60 * 60))
     end,
     min =  function(self)
-        return (math_modf(total_secs(self) / 60))
+        return math_modf(total_secs(self) / 60)
     end,
 }
 
diff --git a/test/app-tap/datetime.test.lua b/test/app-tap/datetime.test.lua
index cf6397679..68d12999d 100755
--- a/test/app-tap/datetime.test.lua
+++ b/test/app-tap/datetime.test.lua
@@ -18,14 +18,14 @@ local MAX_DATE_DAY = 11
 
 local SECS_PER_DAY      = 86400
 local AVERAGE_DAYS_YEAR = 365.25
-local AVERAGE_WEEK_YEAR = (AVERAGE_DAYS_YEAR / 7)
-local MAX_YEAR_RANGE = (MAX_DATE_YEAR - MIN_DATE_YEAR)
-local MAX_MONTH_RANGE = (MAX_YEAR_RANGE * 12)
-local MAX_WEEK_RANGE = (MAX_YEAR_RANGE * AVERAGE_WEEK_YEAR)
-local MAX_DAY_RANGE = (MAX_YEAR_RANGE * AVERAGE_DAYS_YEAR)
-local MAX_HOUR_RANGE = (MAX_DAY_RANGE * 24)
-local MAX_MIN_RANGE = (MAX_HOUR_RANGE * 60)
-local MAX_SEC_RANGE = (MAX_DAY_RANGE * SECS_PER_DAY)
+local AVERAGE_WEEK_YEAR = AVERAGE_DAYS_YEAR / 7
+local MAX_YEAR_RANGE = MAX_DATE_YEAR - MIN_DATE_YEAR
+local MAX_MONTH_RANGE = MAX_YEAR_RANGE * 12
+local MAX_WEEK_RANGE = MAX_YEAR_RANGE * AVERAGE_WEEK_YEAR
+local MAX_DAY_RANGE = MAX_YEAR_RANGE * AVERAGE_DAYS_YEAR
+local MAX_HOUR_RANGE = MAX_DAY_RANGE * 24
+local MAX_MIN_RANGE = MAX_HOUR_RANGE * 60
+local MAX_SEC_RANGE = MAX_DAY_RANGE * SECS_PER_DAY
 
 local incompat_types = 'incompatible types for datetime comparison'
 local incompat_int_types = 'incompatible types for interval comparison'
```